sources/web/datalab/polymer/components/resizable-divider/resizable-divider.ts

```diff
@@ -79,40 +99,77 @@ class ResizableDividerElement extends Polymer.Element {
 
     // stop styling the handle as active because dragging is done
     this.$.divider.classList.remove('active');
+    this._lastMouseUpPosition = this.dividerPosition;
   }
 
   _mouseMoveHandler(event: MouseEvent) {
-    this._resizePanes(event.clientX);
+    const container = this.$.container as HTMLDivElement;
+    const containerRect = container.getBoundingClientRect();
+    const newLeftPaneWidth = event.clientX - containerRect.left;
+    const fullWidthLessDivider = containerRect.width - this._dividerWidth;
+    const newPosition = newLeftPaneWidth * 100 / fullWidthLessDivider;
+    this.dividerPosition =
+        (newPosition < 0) ? 0 : (newPosition > 100) ? 100 : newPosition;
+    // Let observer call _dividerMoved
+  }
+
+  /**
+   * Calculate the new divider position after hideRight changes.
+   */
+  _hideChanged() {
+    if (!this._readyDone) {
```

Comment:
nit: Without using a variable, you could also check that the first argument is undefined, which is the case when the observer callback is called because of initial property setters.

Reply:
Do you mean, add an argument such as _hideChanged(foo)? That doesn't appear to work, I never see an undefined value.

Reply:
Sorry, I meant the second argument. See https://www.polymer-project.org/2.0/docs/devguide/observers#change-callbacks. The second argument is old value, and the first thing done on element init is to set the property to some default value, which will end up calling the observer with `(newValue = default, oldValue = undefined)`.